src/Microsoft.Data.Analysis/PrimitiveDataFrameColumn.cs

```diff
@@ -551,25 +551,41 @@ protected override DataFrameColumn FilterImplementation<U>(U lower, U upper)
                 throw new ArgumentException(string.Format(Strings.MismatchedValueType, typeof(T)), nameof(U));
         }
 
-        public override DataFrame Description()
-        {
-            DataFrame ret = new DataFrame();
-            StringDataFrameColumn stringColumn = new StringDataFrameColumn("Description", 0);
-            stringColumn.Append("Length");
-            stringColumn.Append("Max");
-            stringColumn.Append("Min");
-            stringColumn.Append("Mean");
-            float max = (float)Convert.ChangeType(Max(), typeof(float));
-            float min = (float)Convert.ChangeType(Min(), typeof(float));
-            float mean = (float)Convert.ChangeType(Sum(), typeof(float)) / Length;
+        public override void Description(DataFrame dataFrame)
```

Comment:
Why not have this method return a column instead? And then the caller can add it to a DataFrame or do whatever they want with it.

Reply:
Done.